.github/workflows/check-requirements.yml

```
@@ -0,0 +1,47 @@
+          
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          
+      - name: Get installed packages
+        id: installed-packages
+        run: |
+          pip freeze > installed_packages.txt
+          echo "INSTALLED_PACKAGES=$(cat installed_packages.txt)" >> $GITHUB_ENV
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)